pkg/dashboards/types.go

```diff
@@ -403,7 +424,7 @@ type DashboardVariableDefaultItem struct {
 // DashboardVariableDefaultItemInput - Represents a possible default value item.
 type DashboardVariableDefaultItemInput struct {
 	// The value of this default item.
-	Value DashboardVariableDefaultValueInput `json:"value,omitempty"`
+	Value *DashboardVariableDefaultValueInput `json:"value,omitempty"`
```

Comment:
```suggestion
	Value DashboardVariableDefaultValueInput `json:"value,omitempty"`
```

Reply:
Upon seeing all the above changes I realised there is a gap between tutone generation and manual modification. I will check if these manual changes need to be accommodated